test/e2e/apimachinery/webhook.go

```diff
@@ -1940,12 +1944,18 @@ func testSlowWebhookTimeoutFailEarly(f *framework.Framework) {
 	name := "e2e-test-slow-webhook-configmap"
 	_, err := client.CoreV1().ConfigMaps(f.Namespace.Name).Create(&v1.ConfigMap{ObjectMeta: metav1.ObjectMeta{Name: name}})
 	framework.ExpectError(err, "create configmap in namespace %s should have timed-out reaching slow webhook", f.Namespace.Name)
-	expectedErrMsg := `/always-allow-delay-5s?timeout=1s: context deadline exceeded`
-	if !strings.Contains(err.Error(), expectedErrMsg) {
-		e2elog.Failf("expect error contains %q, got %q", expectedErrMsg, err.Error())
+	isHTTPTimeout := strings.Contains(err.Error(), slowWebhookHTTPTimeoutMsg)
```

Comment:
might be cleaner to check both of  these separately:
* message contains `/always-allow-delay-5s?timeout=1s` (ensures the error came from speaking to the webhook)
* message contains either `context deadline exceeded` or `timeout`

that lets us expand the timeout detecting messages if needed

Reply:
updated